xpk.py

```diff
@@ -1148,32 +1151,51 @@ def create_cluster_configmap(args, system):
   Returns:
     0 if successful and 1 otherwise.
   """
+  configmap_yml = {}
   device_type = args.tpu_type if args.tpu_type else args.device_type
-  data = f'{device_type}: "{int(args.num_slices) * system.vms_per_slice}"'
-  yml_string = cluster_configmap_yaml.format(args=args,
-                                           data=data)
-  tmp = write_temporary_file(yml_string)
-  command = f'kubectl apply -f {str(tmp.file.name)}'
+  resources_data = f'{device_type}: "{int(args.num_slices) * system.vms_per_slice}"'
```

Comment:
nit to add two comments to explain that there are two config maps being created. One on ln 1156 and another on ln 1163.

Reply:
Done!